runtime/spring/src/main/java/com/flit/runtime/spring/FlitExceptionHandler.java

```diff
@@ -11,54 +15,67 @@
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
-import javax.servlet.http.HttpServletRequest;
-import java.util.HashMap;
-import java.util.Map;
-
 @ControllerAdvice
 @Component
 public class FlitExceptionHandler extends ResponseEntityExceptionHandler {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(FlitExceptionHandler.class);
```

Comment:
As much as I prefer 2 spaces for indentation, can we revert these indentation changes and respect the original project indentation conventions? 😄

Reply:
sorry,  IntelliJ automatically formatted it. Let me fix that back